### PR TITLE
[TKW] Test multiple igemm layouts

### PR DESCRIPTION
**Pull request by Hardcode84**
* Test `nchw_fchw` and `nhwc_hwcf` igemm conv layouts.
* Perf test will use `nhwc_hwcf` as IREE seems to produce the best result for it.

### CODE REVIEW

File: tests/kernel/wave/wave_e2e_test.py [hunk 1]
```diff
@@ -984,6 +876,7 @@ def repeat(acc: tkl.Register[M, NF, tkl.f32]) -> tkl.Register[M, NF, tkl.f32]:
         run_bench=run_bench,
         run_config=config,
     ):
+        out = torch.zeros_like(out_ref)
```
Comment (harsh-nod) [inline tests/kernel/wave/wave_e2e_test.py:879]: why is this needed?

**Review by harsh-nod — APPROVED** [review]
lgtm!

File: tests/kernel/wave/wave_e2e_test.py (hunk 1 above)
Comment (Hardcode84) [reply tests/kernel/wave/wave_e2e_test.py:879]: just moved the var closer to it usage
